cpp/src/las/laz_config.cpp

```diff
@@ -43,12 +43,4 @@ LazConfigWriter::LazConfigWriter(const int8_t &point_format_id, const Vector3 &s
         throw std::runtime_error("LasConfig: Supported point formats are 6 to 8.");
 }
 
-// Copy constructor from CopcConfig
-LazConfigWriter::LazConfigWriter(const copc::CopcConfig &copc_config)
```

Comment:
We wouldnt want this anymore?

Reply:
It should be handled implicitly because CopcConfig is a subclass of LazConfig

Reply:
added